adcc/testdata/static_data.py

```diff
@@ -67,4 +78,18 @@
     H        3.8622600497      3.6616656215     -1.2676471822
     H        5.9395792877      1.1934754318     -2.1292489119
     """,  # (R)-2-Methyloxirane
+    #
+    "formaldehyde": """
+    C 2.0092420208996 3.8300915804899 0.8199294419789
+    O 2.1078857690998 2.0406638776593 2.1812021228452
+    H 2.0682421748693 5.7438044586615 1.5798996515014
+    H 1.8588483602149 3.6361694243085 -1.2192956060942
+    """
 }
+
+_potential_files_relative = {
+    "fa_6w": "potentials/fa_6w.pot",  # Formaldehyde + 6 Water molecules
```

Comment:
why not just `os.path.join(thisdir, "potentials/fa_6w.pot")` and skip the fullfile function completely?

Reply:
Done.